utility/log.py

```diff
@@ -14,14 +14,25 @@
 
     - local file
     - logstash server
+
+Initial Log format will be "datetime - level - message"
```

Comment:
Generic comment, Please every logger reletad methods/defintions/objects to this file to a common module.